tests/components/zwave_mqtt/common.py

```diff
@@ -0,0 +1,70 @@
+"""Helpers for tests."""
+import json
+
+from asynctest import Mock, patch
+
+from homeassistant import config_entries, core as ha
+from homeassistant.components.zwave_mqtt.const import DOMAIN
+
+from tests.common import MockConfigEntry
+
+
+async def setup_zwave(hass, entry=None, fixture=None):
+    """Set up Z-Wave and load a dump."""
+    hass.config.components.add("mqtt")
+
+    if entry is None:
+        entry = MockConfigEntry(
+            domain=DOMAIN,
+            title="Z-Wave",
+            connection_class=config_entries.CONN_CLASS_LOCAL_PUSH,
+        )
+
+        entry.add_to_hass(hass)
+
+    with patch("homeassistant.components.mqtt.async_subscribe") as mock_subscribe:
+        await hass.config_entries.async_setup(entry.entry_id)
```

Comment:
This feels weird. Why don't we do here `await async_setup_component(hass, "zwave", {})` ? 

(I think I wrote this code btw, and I don't remember)

Reply:
let's also assert it
```suggestion
        assert await hass.config_entries.async_setup(entry.entry_id)
```

Reply:
Setting up the entry will set up the component. There's no config but we might want to tailor the config entry, so it makes more sense to use that to set up the integration.